e2e/globals.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+const path = require('path')
+const chromedriver = require('chromedriver')
+const request = require('request')
+const config = require('./config/index')
+
+const SCREENSHOT_PATH = './e2e/tests/failure-screenshots/'
+
+const getFileName = testName => {
+  const name = testName.replace(/ /g, '_')
+  return path.resolve(`${SCREENSHOT_PATH}${name}.png`)
+}
+
+let counter = 0
+
+const healthCheck = done => {
+  console.log(`${counter}/20 Health Checking the UI Container...`)
+  request(config.healthCheckUrl, (err, response, body) => {
+    if (!body && counter < 20) {
+      counter += 1
+      setTimeout(healthCheck, 6000, done)
+    } else {
+      done()
+    }
+  })
+}
```

Comment:
The `healthCheck` seems to be assuming that the e2e tests are also starting up a server, so its waiting for the server to be fully started. I'd recommend removing this part if we don't need it yet.

Reply:
Not quite sure why it is needed it, but the tests don't run unless we perform the health check immediately after starting the chromedriver.

Either way, I'd prefer to keep it because if I start seeing the healthCheck loop messages on my console I'm quickly reminded that it's most likely because I've forgotten to run `yarn dev` on my other tab.

Reply:
I see. Ok you convinced me!